changelog.d/6013.bugfix

```diff
@@ -0,0 +1 @@
+Fix v2 identity server calls using `id_access_token` instead of `access_token`, and use HTTP Authorization headers to send them instead of query parameters or JSON bodies.
```

Comment:
Do we want this to have the same changelog as the broken PR so that they're merged in the changelog?

Reply:
Yes that's probably a good idea.